nucleus/security/core/pom.xml

```diff
@@ -103,6 +103,11 @@
             <groupId>org.glassfish.gmbal</groupId>
             <artifactId>gmbal</artifactId>
         </dependency>
+		<dependency>
```

Comment:
Formatting ;)

Reply:
Maybe we need to install a check style rule for tabs in pom.xml files? Somehow a tab got inserted again :O